tests/sentry/tasks/test_derive_code_mappings.py

```diff
@@ -13,7 +13,6 @@
 from sentry.tasks.derive_code_mappings import derive_code_mappings, identify_stacktrace_paths
 from sentry.testutils import TestCase
 from sentry.testutils.helpers import with_feature
-from sentry.testutils.helpers.datetime import before_now, iso_format
```

Comment:
We were adding a lot of metadata for Python events but our code doesn't care about it.